mffpy/reader.py

```diff
@@ -60,6 +60,25 @@ class `Reader` is the main entry point to `mffpy`'s functionality.
     def __init__(self, filename: str):
         self.directory = get_directory(filename)
 
+    @cached_property
+    def mff_flavor(self) -> str:
+        """
+        ```python
+        Reader.mff_flavor
+        ```
+        return flavor of the MFF
+
+        Return string value with the flavor of the MFF either, 'continuous',
+        'segmented', or 'averaged'. This is determined from the entries in
+        the `history.xml` file. If no `history.xml` return 'continuous'.
+        """
+        try:
+            with self.directory.filepointer('history') as fp:
+                history = XML.from_file(fp)
+                return history.mff_flavor()
```

Comment:
[required] Are we really sure that the .mff is continuous if reading history returns a value error?  Can you instead:
* check for the presence of the history file: not present -> return 'continuous'
* if present -> return mff_flavor(), but don't catch any errors

This approach would guard us to attempt to read .mffs that are _ill_.

Reply:
Yeah, I think this is a good idea.